torch/csrc/lazy/core/config.cpp

```diff
@@ -27,6 +27,11 @@ C10_DEFINE_bool(
     false,
     "Use thread pool to schedule backend execution");
 
+C10_DEFINE_bool(
```

Comment:
would it easier to just set `max_cache_size_ ` to 0(I am not sure if there is a way to configure that.)? The code below is doing the exact same thing.

Reply:
this way you don't need to introduce a new flag.

Reply:
What you suggest is 100% doable and a global exists for that purpose.

The intention here is to allow two usages to co-exist - a cache hidden mode needed for use with torch_neuronx.trace (which needs the cache size zero) and XLA device usage where the cache is desirable.

This allows users to interchange XLA and trace usages without having to recreate/refill the cache.  If we think the cache performance benefits are small we could go this route, but this way allows us to hide and restore the cache.

Reply:
ok so you want to enable/disable cache during runtime..